internal/seclang/directives.go

```diff
@@ -229,7 +229,7 @@ func directiveSecResponseBodyAccess(options *DirectiveOptions) error {
 }
 
 // Description: Configures the maximum request body size Coraza will accept for buffering.
-// Default: 134217728 (131072 KB)
+// Default: 1073741824 (1024 MiB)
```

Comment:
See https://github.com/corazawaf/coraza/blob/4b1b82d97ce850c72cb12429f6610530242d2cdc/internal/corazawaf/waf.go#L278

Reply:
Some confusion here. _1gb is **the hard limit** and not the default one. Maybe it is better to set to -1 and fail if not set? cc @anuraaga

Reply:
Since https://github.com/corazawaf/coraza/commit/6482ab129ab66fcc8ac6b5d93ce0febe63d41d56, seems like it is effectively not just the hard limit, but also the default one. Here I wanted to at least align the doc with the implementation.
I agree that it is not the best default value: I'm okay with both setting the default value to the one that users are used from ModSec, or making it a requirement.

This is basically the evolution of https://github.com/corazawaf/coraza/pull/612#discussion_r1097762691

Reply:
Yeah, it is the default one **without applying the recommended conf**. I think making body limit so huge could be problematic so maybe we align the documented default values with the ones in code? cc @anuraaga @jptosso

Reply:
I agree about aligning code to doc